tsconfig.json

```diff
@@ -12,7 +12,7 @@
     "sourceMap": true,
     "target": "es2019",
     "lib": [
-      "es2019",
+      "ESNext",
```

Comment:
why?

Reply:
I added this library in order to have access to the latest JS features (like String.replaceAll() )